app/controllers/form_controller.rb

```diff
@@ -86,7 +86,7 @@ def responses_for_page(page)
         year = params[@log.model_name.param_key]["#{question.id}(1i)"]
         next unless [day, month, year].any?(&:present?)
 
-        result[question.id] = if Date.valid_date?(year.to_i, month.to_i, day.to_i) && year.to_i.between?(2000, 2200)
+        result[question.id] = if Date.valid_date?(year.to_i, month.to_i, day.to_i) && year.to_i.positive?
```

Comment:
For soft validation, we want to retain the correct date where a valid but suspicious date is input.